server/plugins/CMakeLists.txt

```diff
@@ -179,6 +180,7 @@ if (SUPERNOVA)
 	foreach(plugin ${plugin_sources})
 		string(REPLACE .cpp "_supernova" plugin_name ${plugin} )
 		add_library(${plugin_name} MODULE ${plugin})
+		target_link_libraries(${plugin_name} PRIVATE Boost::headers)
```

Comment:
isn't this already covered by line 70?

Reply:
Ah, I don't think this is covered by line 70. This is a separate entry for supernova plugins.